ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -664,6 +664,36 @@ FAN_SS_AL:
   - schedule_run_command
   - control_status
 
+FAN_VSC_SRC:
+  description: "Basic Exhaust fan with start/stop, speed sensor, alarms, scheduled run"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
+  implements:
+  - FAN
+  - VSC
+  - SRC
+
+FAN_VSC_SRC_DPM:
+  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run, dampers"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
+  implements:
+  - FAN
+  - VSC
+  - SRC
+  - DPM
+
+FAN_PS:
```

Comment:
this fan doesnt have anything other than an air pressure sensor? we should model this as another type of fan like FAN_SS and add air pressure sensor as optional.

Reply:
Noted sir. will do.